TOT.Entities/Policy_Entities/TimeOffPolicy.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Collections.Generic;
+using TOT.Entities.Request_Entities;
+
+namespace TOT.Entities.Policy_Entities
+{
+    public class TimeOffPolicy
```

Comment:
Как я писал в https://github.com/ASbeletsky/TimeOffTracker/issues/8#issuecomment-431662507 я предлагаю 
 
```suggestion
    public class EmployeePositionTimeOffPolicy
    {
          public int Id { get; set; }
          public EmployeePosition EmployeePosition { get; set; }
          public TimeOffPolicy Policy { get; set; }
          public ICollection<TimeOffPolicyApproval> Approvals { get; set; }  
    }
```